test/config/imageservice_config.json

```diff
@@ -5,7 +5,7 @@
         "osname": "CentOS",
         "version": "7.0",
         "linktype": "http",
-        "url": "http://buildlogs.centos.org/rolling/7/isos/x86_64/CentOS-7-x86_64-DVD.iso"
+        "url": "http://10.62.59.209/static/images/ubuntu-16.04-server-amd64.iso"
```

Comment:
I think we can use another imageservice_config.json  instead of this one in github. There is no need to modify the original on in github